etc/profile.d/baseimage.sh

```diff
@@ -16,13 +16,28 @@ alias pip3="pip3 --no-cache-dir"
 alias pylint="pylint3"
 alias python="python3"
 
+# Localization
+export LANG="C.UTF-8"
+export LC_ALL="C.UTF-8"
+export LC_CTYPE="C.UTF-8"
+
 # Flask
 export FLASK_APP="application.py"
 export FLASK_DEBUG="0"
 export FLASK_ENV="development"
 
+export APPLICATION_ENV="dev"
```

Comment:
This might break `cs50/server` by putting it into dev mode even in production? Can we move all of these to `cs50/ide` for now?

Reply:
Removed.

Reply:
Added to cs50/ide-docker per https://github.com/cs50/ide-docker/pull/36.